### ISSUE TITLE
Regarding WowUp

### ISSUE THREAD

**Issue by MajSpork** [Vicious-wow/XIV_Databar#29]
I first want to appreciate you making the effort to tag the release to make it work with WowUp. That said I cannot for the life of me figure out how to get it added. I'm still getting the "Addon installation failed: No Release Assets found" message whenever I try to add it. I know you said you CBA if it doesn't work, I just wanna make sure it's not just me being a dummy and using the wrong URL.
![image](https://user-images.githubusercontent.com/69874070/102251352-afe4e500-3ed2-11eb-9c36-00f5694206eb.png)

**Comment by thomasthebear** [comment]
Most likely has something to do with the fact that the repository still only has a `Source Code (zip)` & `Source Code (tar.gz)` available in the Assets of the `v3.0.0` Release, _**though it's just my guess**_. You've got the correct URL though, so no issues there. 

I’ve checked the download and it seems to be all in order, the `.TOC` doesn’t seem to have any noticeable issues & references all assets it needs. Same goes for the `Embeds.xml` & `Core.lua` files. To be perfectly honest, I think it’s just what I stated above, as [**Ajour**](https://github.com/casperstorm/ajour) is even having an issue trying to pull the Addon in. That or the Addon Managers are being very specific in what the files need to be named to pull or they’ve recently updated something on **Github’s** end that has broken the downloads through the Managers. 

Would be much, _**much**_ easier to just download the `Source Code (zip)`, extract the contents so you have a **XIV_Databar-3.0.0** folder and put that inside **World of Warcraft’s Addon** Folder, usually located in the directory shown below _(though I know it can be a hassle to keep updated, but for now it’s probably the best option)_:

>`C:\Program Files (x86)\World of Warcraft\_retail_\Interface\Addons\`

---

> Upon looking into it though, I've noticed that in [**Ajour's**](https://github.com/casperstorm/ajour) case, it's looking for a `Release (zip)` for a given Repository at the URL i've added below _(using XIV_Databar's repository as an example along with the error given from [**Ajour**](https://github.com/casperstorm/ajour)_):
> 
>```diff
>- No zip available for https://api.github.com/repos/Vicious-wow/XIV_Databar/releases/latest
>```
>    
> There is a difference between a `Source Code (zip)` and a `Release (Latest)` in terms of URL, difference being the later subdomains as shown below _(using **https://github.com/Vicious-wow/XIV_Databar** as the basis)_:
>```diff
> ! /archive/v3.0.0.zip  <-- Source Code (zip)
>```
>  TO
>```diff
> + /releases/download/v3.0.0/XIV_Databar-3.0.0.zip  <-- Release (Latest) if Done Properly, though that file doesn't exist currently on the repo
>```
> ---
<br>
<br>
<sup> I am only trying to help out when I have the time, though this went a bit off the rails in terms of a straight forward reply to you (**[MajSpork](https://github.com/MajSpork)**) & ended up being more of a _"Why this is happening and what needs to be changed"_ answer lol. So apologies in advance! Hopefully this will help someone, if not yourself, out in the long run. 😄</sup>

**Comment by Vicious-wow** [comment]
I won't look up what needs to be done for WoWUp to be able to auto-update off of github. Tell me exactly what needs doing, in detail, without any need for me to look up anything myself, and i'll do it. Otherwise i cba.

**Comment by Geodeathdeathgeo** [comment]
This is directly from wowups user guide.

GitHub Notes

Not all GitHub repositories will work with the import process. In order for WowUp to be able to install an addon from GitHub the author must have created a tagged release. That tagged release must then contain a packaged zip file that WowUp can find.

A good example of a working GitHub repo is Teelo's Altoholic Retail. 

---------
For wowup to recognize the addon from the url there needs to be actual zipped file attached to the tagged release like what Altoholics does on github.

Basically make a zipped file of the version you are releasing, attach it to the tagged release and wowup should recognize it for download. Otherwise no auto-update. It doesn't recognize the source code zip, that is why its not working.

I only made this account to comment here, and have no idea how to use github. Just wanted to try and help with the situation.

**Comment by Vicious-wow** [comment]
Let me know if it works now. Unless there's some magic behind the scenes going on this should make it happen.

**Comment by MajSpork** [comment]
> Let me know if it works now. Unless there's some magic behind the scenes going on this should make it happen.

Can confirm it works. Thank you so much!
